test/24_device_additions.ts

```diff
@@ -27,6 +27,17 @@ export default () => {
 		describe('Device additions', function () {
 			let pineUser: PineTest;
 			const ctx: AnyObject = {};
+
+			const itShouldHaveOverallStatus = (
+				testCase: string,
+				deviceName: string,
+				overallStatus: string,
+			) => {
+				it(testCase.replace('${OVERALL_STATUS}', overallStatus), async () => {
```

Comment:
NIT:
```suggestion

			const itShouldHaveOverallStatus = (
				overallStatus: string,
				restTitle: string,
				deviceName: string,
			) => {
				it(`should have a(n) ${overallStatus} ${restTitle}, async () => {
```
This way the test case would naturally read v :)
```
					itShouldHaveOverallStatus(
						versions.gt(version, 'v6') ? 'reduced-functionality' : 'idle',
						'when the heartbeat is Offline',
						'deviceVpnOnHeartbeatOffline',
					);
```